test-tables.md

```diff
@@ -0,0 +1,28 @@
+# Table Sorting Test
+
+This file tests the new table sorting functionality in the Agregore renderer.
+
+## Sample Data Table
```

Comment:
These tables should get moved to the `theme/example` file in the browser proper: https://github.com/AgregoreWeb/agregore-browser/blob/master/app/pages/theme/example.md